tools/python/m2k-osc/ain_tab.py

```diff
@@ -0,0 +1,199 @@
+import matplotlib
+import numpy as np
+import tkinter as tk
+from tkinter import ttk
+import matplotlib.figure as  fg
+import libm2k
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib import style
+import threading
+import time
+import ctx_tab
+import aout_tab
+LARGE_FONT = ("Verdana",12)
+SMALL_FONT = ("Vedana",10)
+
+samples_queuech0=[]
```

Comment:
[curios] generally speaking about this Python module;
is it a standalone app or a library?

depending on the case, it's a good idea to keep samples queued on the module or not;

Reply:
it's supposed to be stand-alone app to showcase functions of the libm2k library